transport/BaseUdpEndpoint.h

```diff
@@ -71,6 +71,18 @@ class BaseUdpEndpoint : public Endpoint, RtcePoll::IEventListener
         memory::UniquePacket packet;
     };
 
+    struct RateMetrics
+    {
+        utils::TrackerWithSnapshot<10, utils::Time::ms * 100, utils::Time::sec> receiveTracker;
+        utils::TrackerWithSnapshot<10, utils::Time::ms * 100, utils::Time::sec> sendTracker;
+        EndpointMetrics toEndpointMetrics(size_t size) const
+        {
+            return EndpointMetrics(size,
```

Comment:
should we rename it queueSize to be more specific what is stored in EndpointMetrics.

Reply:
Done.